blog/templates/blog/CLIP_form.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'blog/base.html' %} 
+{% block content %} 
+    <h2>New CLIP Submission</h2>
+    <form method="POST" class="post-form">{% csrf_token %} 
+        {{ form.as_p }}
+        <p>Add fastq</p>
+        fastq name<input type="text" name="fastq_title">
+        fastq path<input type="text" name="fastq_path">
+        adapter path<input type="text" name="adapter_path">
+        <ul>
+            {% for fastq in fastq %} 
+                {% if fastq.complete == True %} <!-- Not sure what complete means? -->
```

Comment:
We can probably change 'complete' to 'checked', I was just following the tutorial closely to test it first.

Reply:
I don't think I needed 'complete' in order to save the 'checked' field, but okay sounds good. Let's discuss tomorrow whether or not to keep this in.